zcash_transparent/src/bundle.rs

```diff
@@ -262,13 +266,13 @@ pub struct TxOut {
     #[deprecated(since = "0.4.1", note = "use the value() accessor instead.")]
     pub value: Zatoshis,
     #[deprecated(since = "0.4.1", note = "use the script_pubkey() accessor instead.")]
-    pub script_pubkey: Script,
+    pub script_pubkey: script::PubKey,
```

Comment:
My previous comment implies that we cannot use `script::PubKey` here. We either need to use `script::Component<PossiblyBad>` (which is currently impossible because `PossiblyBad` is not exposed in the public API), or just not parse the script at this point at all, and keep it as a `Vec<u8>` (effectively what we did before).